cmd/juju/application/deploy.go

```diff
@@ -71,6 +72,7 @@ type ApplicationAPI interface {
 	SetConstraints(application string, constraints constraints.Value) error
 	Update(apiparams.ApplicationUpdate) error
 	ScaleApplication(application.ScaleApplicationParams) (apiparams.ScaleApplicationResult, error)
+	Consume(arg crossmodel.ConsumeApplicationArgs) (string, error)
```

Comment:
It's interesting that this call is not part of the offer API

Reply:
I know right :| That took sometime to work out!